content/en/docs/pipelines/installation/overview.md

```diff
@@ -21,6 +21,9 @@ all Kubeflow components and more integration with each platform.
 1. Do you want to use other Kubeflow components in addition to Pipelines?
 
     If yes, choose the [full Kubeflow deployment](#full-kubeflow-deployment).
+1. Can you use a cloud/on-prem Kubernetes cluster?
+
+    If no, you can try Kubeflow Pipelines on a local Kubernetes cluster following [Deploying Kubeflow Pipelines on a local cluster](/docs/pipelines/installation/localcluster-deployment) for learning and testing purposes.
```

Comment:
```suggestion
    If you can't, you should try using Kubeflow Pipelines on a local Kubernetes cluster for learning and testing purposes by following the steps in [Deploying Kubeflow Pipelines on a local cluster](/docs/pipelines/installation/localcluster-deployment).
```

WDYT? cc @alfsuse @RFMVasconcelos 

Thanks @Bobgy

Reply:
LGTM! Let's wait for their opinions too

Reply:
/lgtm sorry I was not much very active these days.